.env

```diff
@@ -1 +1,2 @@
-REACT_APP_API_URL=http://localhost:8000/api/
\ No newline at end of file
+# REACT_APP_API_URL=https://ah-map-staging.herokuapp.com/api/
+REACT_APP_API_URL=http://127.0.0.1:8000/api/
```

Comment:
`.env `  should not be pushed to github. you should add it to `.gitignore`

Reply:
I have removed it, It is already in the .gitignore file.